docs/deployment/config_descriptions.md

```diff
@@ -90,7 +90,7 @@ This table describes the fields that must be present in `config.yml` to successf
 | launchpad.api  | | launchpad API endpoint
 | launchpad.certificate  | launchpad.pfx | the Launchpad PKI certificate pfx file
 | launchpad.passphrase  | | the passphrase of the Launchpad PIK certificate
-| oauth.provider | earthdata | the authentication provider used for Cumulus API, possible values are 'earthdata', 'google', 'launchpad'
+| oauthProvider | earthdata | the authentication provider used for Cumulus API, possible values are 'earthdata', 'google', 'launchpad'
```

Comment:
oauth.provider has been used in backend_api_lambdas.yml.  We should keep this configuration, and may add another parameter inside, e.g. with default value
oauth:
    provider: earthdata
    userGroup: N/A